src/data/Mesh.cpp

```diff
@@ -1088,6 +1088,61 @@ void odc::data::Mesh::new_inimesh(int MEDIASTART,
 #endif
         }
   }
+  else if(MEDIASTART == 4)
```

Comment:
Might be good to comment here about what MEDIASTART 4 is.  

Also at some point max_vse, min_vse etc. need to be reduced over MPI (for all MEDIASTART >= 1).  Not needed for this commit, but leave a TODO so it's not forgotten.